### DISCUSSION TITLE
"winfetch" command not working after install via chocolaty

### DISCUSSION THREAD

**Discussion by cmdshft** [lptstr/winfetch#229] [Q&A]
Hello all. I seem to be unable to run using just "winfetch" after installing via WingetUI and Chocolaty.I get the following error: 

`winfetch: The term 'winfetch' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.`

I am able to run it remotely and my config works. How do I correct the error?

**Comment by JasonTurley** [comment]
Hi, reading the error message, it sounds like `winfetch` is not in your system or current user’s PATH.

**Comment by rashil2000** [comment]
> I am able to run it remotely and my config works. How do I correct the error?

How are you running it remotely?
